lib/commands/deploy/index.js

```diff
@@ -44,7 +44,7 @@ class DeployCommand extends AbstractCommand {
             if (err instanceof CliWarn) {
                 Messenger.getInstance().warn(err.message);
             } else if (err instanceof CliFileNotFoundError) {
-                Messenger.getInstance().warn('Failed to find the ask-resources.json file to deploy with from the current directory.');
+                Messenger.getInstance().warn(err.message);
```

Comment:
Could we add a filePath/fileName property (as a discriminator) to https://github.com/alexa/ask-cli/blob/develop/lib/exceptions/cli-file-not-found-error.js#L6, and then check the value of the filePath here? I believe the err message for ask-resources.json file not found needs this explanation that we need the file to deploy the project. 
It's even better we can explain to users that skill.json is missing in your skill-package component. Great UI in general.

Reply:
May be when we throw the error we just output more specific message? 
https://github.com/alexa/ask-cli/blob/fbb92f2c0a46e6d6e92c7459127ceb24ad867640/lib/model/abstract-config-file.js#L107

Adding discriminator seems to just add another level of complexity.

Reply:
Well we can try this general message idea, since `npm` is also following this pattern and they don't have much issue I think.
```
$ npm install
npm WARN saveError ENOENT: no such file or directory, open '/Users/nongwang/Amazon/brazil/AVSPortal/src/package.json'
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN enoent ENOENT: no such file or directory, open '/Users/nongwang/Amazon/brazil/AVSPortal/src/package.json'
npm WARN src No description
npm WARN src No repository field.
npm WARN src No README data
npm WARN src No license field.
```
```
$ npm publish
npm ERR! path package.json
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open
npm ERR! enoent ENOENT: no such file or directory, open 'package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```
If we can add that ask-cli attempts to open/use this file but it not exists.